### PR TITLE
how to check rails logs

### PR DESCRIPTION
**Pull request by RahulMahale**

### CODE REVIEW

File: docs/how-to.md [hunk 1]
```diff
@@ -37,6 +37,17 @@ $ bundle exec cap staging db:restore
...
+### How to find rails logs
+
+We can use any of the following commands
+
+```
+ bundle exec cap staging "log:tail["*"]" 
+
+ bundle exec cap staging "log:tail["unicorn*"]" 
+
+ bundle exec cap staging "log:tail["staging"]"
```
Comment (neerajdotname) [inline docs/how-to.md]: What does this command do.

**Comment by neerajdotname** [comment]
@RahulMahale Please rebase.

**Comment by RahulMahale** [comment]
@neerajdotname Done, Please review.

**Comment by neerajdotname** [comment]
@RahulMahale how-to file is deleted in master. now we have one file for one title. Please browse docs folder to see how things are setup now. And the final file is index.md . Read the top of that file to see how to run rake command to build index.md.

**Comment by RahulMahale** [comment]
Checked and modified changes accordingly, this is smart thing to manage the docs, +1.

File: docs/how-to-check-rails-logs.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+### How to find rails logs
```
Comment (neerajdotname) [inline docs/how-to-check-rails-logs.md]: There is already a doc for it. Please edit that. https://github.com/bigbinary/boxy/tree/master/docs/rails

**Comment by RahulMahale** [comment]
deleted the new file and modified the file https://github.com/bigbinary/boxy/blob/master/docs/rails/how-to-see-log-when-no-syslog.md

**Pull request merged**